gcp_variant_transforms/libs/bigquery_util.py

```diff
@@ -379,6 +379,8 @@ def compose_table_name(base_name, suffix, is_sample=False):
       SAMPLE_TABLE_SUFFIX_SEPARATOR if is_sample else TABLE_SUFFIX_SEPARATOR)
   return separator.join([base_name, suffix])
 
+def get_table_base_name(table_name):
+  return table_name.split(SAMPLE_TABLE_SUFFIX_SEPARATOR)[0]
```

Comment:
Hmmm, technically this should be TABLE_SUFFIX_SEPARATOR before my follow up PR, but I guess this is fine. No-op.

Reply:
You are right, changed.
My intention was/is to later remove `SAMPLE_TABLE_SUFFIX_SEPARATOR` and change the value of `TABLE_SUFFIX_SEPARATOR` to `__`.